### ISSUE TITLE
4th front page content boxes dropped to newline.

### ISSUE THREAD

**Issue by denzelchia** [denzelchia/t4press-tracker#21]
Happens only in Desktop Safari on front end when using Theme Options -> Front page content boxes.

https://theme4press.com/support/topic/front-page-content-boxes-of-evolve-not-aligned-after-update/#post-26782

**Comment by denzelchia** [comment]
The following is the css fix for all column combinations in Mac Safari.

```
/** adjust margins will fix the layout **/
.home-content-boxes .col-md-3.content-box {
margin:0 0.8% 0 1%;
}
/** adjust margins will fix the layout **/
.home-content-boxes .col-md-4.content-box {
margin:0 0.8% 0 1%;
}
/** adjust margins will fix the layout **/
.home-content-boxes .col-md-6.content-box {
margin:0 0.8% 0 1%;
}	
/** To prevent button from overlapping content **/
.col-md-6 .content-box p{
min-height:200px;
}	
/** To prevent button from overlapping content **/
 .col-md-3 .content-box p,  .col-md-4 .content-box p{
min-height:120px;
}
/**overwrite so as to allow button to center align.**/
.cntbox_btn a{
position:relative;
left:0 !important;
top:0 !important;
}
```
Screen-cast to show the test result on Mac Safari
http://screencast-o-matic.com/watch/cDfr2r1EEs

**Comment by denzelchia** [comment]
Testing custom-css.php and style.css from Romik.
Screen-cast of Frontpage content boxes in different views.
http://screencast-o-matic.com/watch/cDf3jf1GZQ

**Comment by denzelchia** [comment]
Testing custom-css.php from Romik.
For 2 column Frontpage content boxes in different views.
http://screencast-o-matic.com/watch/cDf3jB1GZA
All works properly.

**Comment by denzelchia** [comment]
The following is a zip file containing screenshots of iPad test.
[iPad screenshots.zip](https://github.com/denzelchia/t4press-tracker/files/232802/iPad.screenshots.zip)
 
Testing iPad Views.

Test 4 column

boxed 1600px landscape -> columns ok, button goes to top.
boxed 1600px portrait -> all ok.

boxed 1200px landscape -> columns ok, button goes to top.
boxed 1200px portrait -> all ok.

boxed 985px landscape -> columns ok, button goes to top.
boxed 985px portrait -> all ok.

boxed 800px landscape -> columns ok, button goes to top.
boxed 800px portrait -> all ok.

wide 1600px landscape -> columns ok, button goes to top.
wide 1600px portrait -> all ok.

wide 1200px landscape -> columns ok, button goes to top.
wide 1200px portrait -> all ok.

wide 985px landscape -> columns ok, button goes to top.
wide 985px portrait -> all ok.

wide 800px landscape -> columns ok, button goes to top.
wide 800px portrait -> all ok.

Test 3 column and 2 column ..
Basically all the same.

landscape -> columns ok, button goes to top.
portrait -> all ok.

iPhone portrait and landscape, all single column, stacked up.
Looks ok.
